### PR TITLE
add beautiful-jekyll theme

### PR DESCRIPTION
**Pull request by daattali**

### CODE REVIEW

File: _posts/2015-09-06-beautiful-jekyll.markdown [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: Beautiful Jekyll
+date: 2015-09-06 00:00:00
+homepage: https://github.com/daattali/beautiful-jekyll
+download: https://github.com/daattali/beautiful-jekyll/archive/master.zip
+demo: http://deanattali.com/beautiful-jekyll/
+author: Dean Attali
+thumbnail: beautiful-jekyll.png
+license: MIT
```
Comment (gynter) [inline _posts/2015-09-06-beautiful-jekyll.markdown]: Hello! To assure the similar formatting and style for all themes please change the `MIT` to `MIT License`.

**Comment by gynter** [comment]
Hello and thank You for the submission!

This theme is already part of my 27 new themes PR #181. Is it okay with You that You'd close this PR and I'll update mine with Your specified content?

**Comment by daattali** [comment]
Oh cool, I didn't see that my theme was in an outstanding pull request, yes of course there's no need to duplicate. Thank you!

**Comment by gynter** [comment]
No problem, I added Your author notes to my PR.

**Pull request closed without merging**